src/utils/upload/index.ts

```diff
@@ -0,0 +1,51 @@
+import { FileWithCollection } from 'services/upload/uploadManager';
+import { FileInMemory, MetadataObject } from 'services/upload/uploadService';
+
+const TYPE_JSON = 'json';
+
+export function fileAlreadyInCollection(
+    existingFilesInCollection,
```

Comment:
Is it possible to specify type of `existingFilesInCollection` similar to `newFile:` **FileInMemory**?